backend/app/groups/service.py

```
@@ -95,7 +95,59 @@ async def get_group_by_id(self, group_id: str, user_id: str) -> Optional[dict]:
+                            "user": {
+                                "name": user.get("name", f"User {member_user_id[-4:]}") if user else f"User {member_user_id[-4:]}",
+                                "email": user.get("email", f"{member_user_id}@example.com") if user else f"{member_user_id}@example.com",
+                                "avatar": user.get("imageUrl") or user.get("avatar") if user else None
+                            } if user else {
+                                "name": f"User {member_user_id[-4:]}",
+                                "email": f"{member_user_id}@example.com",
+                                "avatar": None
+                            }
+                        }
+                        enriched_members.append(enriched_member)
+                    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)